docs/sdk/src/reference_docs/runtime_vs_smart_contract.rs

```diff
@@ -1,6 +1,209 @@
-//! Runtime vs. Smart Contracts
+//! # Runtime vs. Smart Contracts
 //!
-//! Notes:
+//! *TL;DR*: If you need to create a *Blockchain*, then write a runtime. If you need to create a
+//! *DApp*, then write a Smart Contract.
```

Comment:
I have had numerous rants about the misuse of the word "DApp" in this space. So, I want to ask the author, what do you mean by DApp throughout this page? 

Specifically, let's look at https://apps.acala.network/. Is this a DApp? Bear in mind that this page is purely interacting with a blockchain runtime, no smart contracts. 

Once we conclude this discussion, it would be good to update the glossary as well.

Reply:
Maybe a bit late, and already resolved on another conversation.

But would love to drop my opinion here anyway.

As I see it DApps are applications with their logic in a decentralized and distributed system, for instance a blockchain. Whether this logic is part of a runtime or a smart contract doesn't play a differentiating role to me.

I'd argue https://staking.polkadot.network/ is a great DApp in the Polkadot ecosystem.